sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager.swift

```diff
@@ -100,6 +113,30 @@ internal final class URLSessionTransferManager: NSObject, TransferManager, URLSe
         return container
     }()
 
+    /// Retrieve all managed transfers created by the transfer manager.
+    internal var transferCollection: TransferCollection {
+        let matching: [BlobTransfer] = StorageBlobClient.manager.transfers.compactMap { $0 as? BlobTransfer }
```

Comment:
Shouldn't this (and the other instances of this line, below) just be `transfers.compactMap`? Since the TM has its list of transfers already.